docs/reference/filebeat/filebeat-input-cel.md

```diff
@@ -557,6 +569,74 @@ stack: ga 8.12.0
 When set to `true`, Digest Authentication challenges are not reused.
 
 
+### `auth.file.enabled` [_auth_file_enabled]
+
+```{applies_to}
+stack: ga 9.3.0
+```
+
+When set to `false`, disables the file auth configuration. Default: `true`.
+
+::::{note}
+File auth settings are disabled if either `enabled` is set to `false` or the `auth.file` section is missing.
+::::
+
+
+### `auth.file.path` [_auth_file_path]
+
+```{applies_to}
+stack: ga 9.3.0
+```
+
+The path to the file containing the authentication value. The file contents are trimmed before use. This field is required when file auth is enabled.
+
+::::{warning}
+By default, Filebeat requires the file to have `0600` permissions (read/write for owner only) and will fail if the file has more permissive permissions. This security measure helps prevent unauthorized access to credentials. To allow files with different permissions, set `relaxed_permissions: true`.
+
+**Note:** On Windows, POSIX-style permission checking is not enforced. Ensure file security using NTFS file permissions or Access Control Lists (ACLs).
```

Comment:
Consider aligning the language with what you included in the `auth.file.relaxed_permissions` description later on the page and linking to that heading. Also, this content is already inside a note-type element so you don't need to include `Note:` in the second paragraph.

```suggestion
By default, Filebeat requires the file to have `0600` permissions (read/write for owner only) and will fail to start if the file is more permissive. This security measure helps prevent unauthorized access to credentials. To allow files with different permissions, set [`relaxed_permissions`](#_auth_file_relaxed_permissions) to `true`.

On Windows, POSIX-style permission checking is not enforced. Ensure file security using NTFS file permissions or Access Control Lists (ACLs).
```